Dockerfile.prod

```diff
@@ -9,13 +9,13 @@ ENV FLASK_APP=hubgrep
 
 WORKDIR /var/task
 
-COPY ./ /var/task/
-
+COPY ./requirements.txt /var/task/requirements.txt
 RUN pip install -r requirements.txt
-
 # unicorn is used to serve the app
 RUN pip install gunicorn
 
+COPY ./ /var/task/
+
```

Comment:
changed the order for faster building

Reply:
var/task contains enough to make a speed difference on installing? I wouldve thought it was egal. =)

Reply:
no - before we copied everything to /var/task, before we pip-installed, which changes the layer in docker, so it would run `pip install` step every time.
now it only runs when requirements.txt changed.